sdk/src/main/java/com/hedera/hashgraph/sdk/Executable.java

```diff
@@ -61,6 +59,9 @@
     static final Pattern RST_STREAM = Pattern
         .compile(".*\\brst[^0-9a-zA-Z]stream\\b.*", Pattern.CASE_INSENSITIVE | Pattern.DOTALL);
 
+    @SuppressWarnings("java:S2245")
+    protected static final Random random = new Random();
```

Comment:
I think you can replace `Random` with `SecureRandom` and remove the warning. [More info](https://wiki.sei.cmu.edu/confluence/pages/viewpage.action?pageId=88487841)